[DOCS-8992] Add helm chart values instructions for connecting Agent to Worker (#25415)
DataDog/documentation: layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent.md

To send Datadog Agent logs to the Observability Pipelines Worker, update your [Agent configuration file][1031] with the following:
```
observability_pipelines_worker:
  logs:
    enabled: true
    url: "http://<OPW_HOST>:8282"
```

`<OPW_HOST>` is the IP/URL of the host (or load balancer) associated with the Observability Pipelines Worker.

For CloudFormation installs, use the `LoadBalancerDNS` CloudFormation output for the URL.

> For Kubernetes installs, the internal DNS record of the Observability Pipelines Worker service can be used. For example: `opw-observability-pipelines-worker.default.svc.cluster.local`.

Reviewer: ```suggestion
For Kubernetes installs, you can use the internal DNS record of the Observability Pipelines Worker service. For example: `opw-observability-pipelines-worker.default.svc.cluster.local`.
```
